src/api/nft.ts

```diff
@@ -30,6 +30,23 @@ export interface NftContract extends BaseNftContract {
   symbol?: string;
   /** The number of NFTs in the contract as an integer string. */
   totalSupply?: string;
+  /** OpenSea's metadata for the contract. */
+  openSea?: OpenSeaCollectionMetadata;
+}
+
+/**
+ * OpenSea's metadata for an NFT collection.
+ */
+export interface OpenSeaCollectionMetadata {
+  floorPrice: number;
```

Comment:
Please mark all of these fields as optional, since they're provided on a best-effort basis.